### ISSUE TITLE
please update repository status

### ISSUE THREAD

**Issue by lduboeuf** [Sturmflut/ubuntu-touch-sdl-template#3]
hello, just searched for `SDL Ubuntu Touch` on the internet and landed here. I guess this template does not build for current release of Ubuntu Touch, am i wrong ? .

**Comment by Sturmflut** [comment]
Hi,

I haven't tried building it since about 2017, and it was already broken back then.

**Comment by lduboeuf** [comment]
yes things have changed a bit since then. David Kozub  made a new template with SDL2 that you can check out here: https://gitlab.com/zub2/SDLDemo
